packages/angular/cli/models/analytics-collector.ts

```diff
@@ -0,0 +1,312 @@
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ *
+ */
+import { analytics } from '@angular-devkit/core';
+import { execSync } from 'child_process';
+import * as debug from 'debug';
+import * as https from 'https';
+import * as os from 'os';
+import * as querystring from 'querystring';
+
+interface BaseParameters extends analytics.CustomDimensionsAndMetricsOptions {
+  [key: string]: string | number | boolean | undefined | (undefined | boolean | number | string)[];
+}
+
+interface ScreenviewParameters extends BaseParameters {
+  /** Screen Name */
+  cd?: string;
+  /** Application Name */
+  an?: string;
+  /** Application Version */
+  av?: string;
+  /** Application ID */
+  aid?: string;
+  /** Application Installer ID */
+  aiid?: string;
+}
+
+interface TimingParameters extends BaseParameters {
+  /** User timing category */
+  utc?: string;
+  /** User timing variable name */
+  utv?: string;
+  /** User timing time */
+  utt?: string | number;
+  /** User timing label */
+  utl?: string;
+}
+
+interface PageviewParameters extends BaseParameters {
+  /**
+   * Document Path
+   * The path portion of the page URL. Should begin with '/'.
+   */
+  dp?: string;
+  /** Document Host Name */
+  dh?: string;
+  /** Document Title */
+  dt?: string;
+  /**
+   * Document location URL
+   * Use this parameter to send the full URL (document location) of the page on which content resides.
+   */
+  dl?: string;
+}
+
+interface EventParameters extends BaseParameters {
+  /** Event Category */
+  ec: string;
+  /** Event Action */
+  ea: string;
+  /** Event Label */
+  el?: string;
+  /**
+   * Event Value
+   * Specifies the event value. Values must be non-negative.
+   */
+  ev?: string | number;
+  p?: string;
+  dp?: string;
+}
+
+/**
+ * See: https://developers.google.com/analytics/devguides/collection/protocol/v1/devguide
+ */
+export class AnalyticsCollector implements analytics.Analytics {
+  private trackingEventsQueue: Record<string, string | number | boolean>[] = [];
+  private readonly parameters: Record<string, string | number | boolean> = {};
+  private readonly analyticsLogDebug = debug('ng:analytics:log');
+
+  constructor(
+    trackingId: string,
+    userId: string,
+  ) {
+    // API Version
+    this.parameters['v'] = '1';
+    // User ID
+    this.parameters['cid'] = userId;
+    // Tracking
+    this.parameters['tid'] = trackingId;
+
+    this.parameters['ds'] = 'cli';
+    this.parameters['ua'] = _buildUserAgentString();
+    this.parameters['ul'] = _getLanguage();
+
+    // @angular/cli with version.
+    const { version, name } = require('../package.json');
+    this.parameters['an'] = name;
+    this.parameters['av'] = version;
+
+    // We use the application ID for the Node version. This should be "node 10.10.0".
+    const nodeVersion = process.release.name || process.argv0 + ' ' + process.version;
```

Comment:
I think the intent was:
```suggestion
    const nodeVersion = (process.release.name || process.argv0) + ' ' + process.version;
```
However, `process.release.name` will always be `node` (https://nodejs.org/api/process.html#process_process_release) so this could probably be reduced to:
```suggestion
    const nodeVersion = `node ${process.version}`;
```
`process.version` has a `v` prefix though (e.g., `v14.16.0`).  If we want to match the comment than `process.versions.node` can be used instead.  It has the same version but no `v` prefix.

Reply:
I think it's best to use `node ${process.version}` to retain the current behaviour.